lib/components/RecentBlocks.js

```diff
@@ -30,32 +43,21 @@ class RecentBlocks extends React.Component {
 
   render() {
     const { getRecentBlocks, recentBlocks, progress } = this.props;
-    // use this to ensure order of columns
-    // otherwise table just uses object keys
-    const colHeaders = [
-      'depth',
-      'height',
-      'hash',
-      'version',
-      'prevBlock',
-      'merkleRoot',
-      'time',
-      'bits',
-      'nonce',
-      'txs'
-    ];
-
     let table;
     if (Array.isArray(recentBlocks) && recentBlocks.length) {
       // structure data for use in the expanded row component
-      recentBlocks.sort((a, b) => a - b);
+      recentBlocks.sort((a, b) => a - b).forEach(block => {
```

Comment:
Could you elaborate on what this code is doing exactly? Why a `forEach` instead of a `map`?

Reply:
map returns a new array, this is just going through each item and updating/reformatting their values. I just didn't think it was necessary to create a new array of blocks to update the values.

I noticed that some coinbase values were coming back as arrays with the hash split into (this seems to be unique to handshake blocks actually, I should check with that team to see if there is a specific reason for it). For display purposes though the coinbase should be represented as a string.

Reply:
You should add comments to the code around the coinbase thing

Reply:
What does subtracting two objects do in the `sort` comparator?

Reply:
Oh yeah, huh. Good question! `git blame` puts that as 4 months ago. That returns NaN though so not sure what I was thinking. Updating to make sure it sorts by height and adding comments.